bufferobserver.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+// BufferObserver separates observer functionality out of file
+type BufferObserver interface {
+	inserted(q0 int, r []rune) // Callback function to update the observer's texts
```

Comment:
Please correct the formatting of your comments to follow `godoc` conventions here and elsewhere.